commit 69a3bcb14950561392b0bb3b0b398ddada3400b7
Author: Derek Foreman <derekf@osg.samsung.com>
Date:   Thu Nov 24 12:11:55 2016 -0600

    drm/vc4: Fix race between page flip completion event and clean-up
    
    There was a small window where a userspace program could submit
    a pageflip after receiving a pageflip completion event yet still
    receive EBUSY.
    
    Signed-off-by: Derek Foreman <derekf@osg.samsung.com>
    Signed-off-by: Eric Anholt <eric@anholt.net>
    Reviewed-by: Eric Anholt <eric@anholt.net>
    Reviewed-by: Daniel Stone <daniels@collabora.com>
    (cherry picked from commit 26fc78f6fef39b9d7a15def5e7e9826ff68303f4)

Index: linuxtest/dummy/rpi_1165_69a3bcb14950561392b0bb3b0b398ddada3400b7.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1165_69a3bcb14950561392b0bb3b0b398ddada3400b7.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
